pom.xml

```diff
@@ -1,99 +1,118 @@
-        <artifactId>nexus-plugins</artifactId>
-        <version>3.13.0-01</version>
-    </parent>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
     <groupId>org.sonatype.nexus.plugins</groupId>
-    <artifactId>nexus-repository-composer</artifactId>
-    <name>${project.groupId}:${project.artifactId}</name>
-    <version>0.0.2</version>
-    <inceptionYear>2018</inceptionYear>
-    <packaging>bundle</packaging>
-
-    <dependencies>
-
-        <dependency>
-            <groupId>org.sonatype.nexus</groupId>
-            <artifactId>nexus-plugin-api</artifactId>
-            <scope>provided</scope>
-        </dependency>
-
-        <dependency>
-            <groupId>org.sonatype.nexus</groupId>
-            <artifactId>nexus-repository</artifactId>
-            <scope>provided</scope>
-        </dependency>
-
-        <dependency>
-            <groupId>org.apache.commons</groupId>
-            <artifactId>commons-compress</artifactId>
-        </dependency>
-
-        <dependency>
-            <groupId>org.sonatype.goodies</groupId>
-            <artifactId>goodies-testsupport</artifactId>
-            <scope>test</scope>
-        </dependency>
-
-        <dependency>
-            <groupId>org.skyscreamer</groupId>
-            <artifactId>jsonassert</artifactId>
-            <version>1.5.0</version>
-            <scope>test</scope>
-        </dependency>
-
-        <dependency>
-            <groupId>org.sonatype.nexus</groupId>
-            <artifactId>nexus-rapture</artifactId>
-            <scope>provided</scope>
-        </dependency>
-        
-    </dependencies>
-
-    <build>
+    <artifactId>nexus-plugins</artifactId>
+    <version>3.19.1-01</version>
+  </parent>
+  <artifactId>composer-parent</artifactId>
+  <version>0.0.2</version>
+  <packaging>pom</packaging>
+  <name>${project.groupId}:${project.artifactId}-parent</name>
+  <inceptionYear>2018</inceptionYear>
+  <modules>
+    <module>nexus-repository-composer</module>
+    <module>nexus-repository-composer-it</module>
+  </modules>
+  <distributionManagement>
+    <snapshotRepository>
+      <id>ossrh</id>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+    </snapshotRepository>
+  </distributionManagement>
+  <properties>
+    <nxrm-version>3.19.1-01</nxrm-version>
+  </properties>
+  <repositories>
+    <!-- ensure we can find the parent pom when starting from an empty local .m2 repository -->
+    <repository>
+      <id>rso-snapshots</id>
+      <url>https://repository.sonatype.org/content/repositories/snapshots</url>
+    </repository>
+  </repositories>
+  <build>
+    <plugins>
+      <!--
+          Ensure use of Java 8, as required by NXRM
+          -->
+      <plugin>
+        <artifactId>maven-enforcer-plugin</artifactId>
```

Comment:
Thanks for this change. With this change in place, I now get the helpful error message below when I accidentally try to build with a "too new" java version (as opposed to an odd cryptic totally unrelated message):

```
[WARNING] Rule 0: org.apache.maven.plugins.enforcer.RequireJavaVersion failed with message:
Detected JDK Version: 11.0.4 is not in the allowed range [1.8,1.9).
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for org.sonatype.nexus.plugins:composer-parent-parent 0.0.2:
[INFO] 
[INFO] org.sonatype.nexus.plugins:composer-parent-parent .. FAILURE [  0.244 s]
```